src/components/Export.tsx

```diff
@@ -501,6 +581,8 @@ const ExportDialog: ExportDialogComponent = ({
               gitHubAccessToken={gitHubAccessToken}
               setGitHubAccessToken={setGitHubAccessToken}
               setCanSendToGitHub={setCanSendToGitHub}
+              githubDestination={githubDestination}
+              setGithubDestination={setGithubDestination}
```

Comment:
[nit] in general, use case specific state/props and callbacks should be defined within use case specific components. is there a way we could push all of this state within the component instead?

Reply:
We could store these in `localStorage` and do a check `onClick` for these values.  But if we do that for `setCanSendToGitHub` we would lose the state dependent UI of a disabled `Export` button.

Reply:
https://github.com/RoamJS/query-builder/pull/255